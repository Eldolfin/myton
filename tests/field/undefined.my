class Foo:
foo = Foo()

foo.bar # expect runtime error: Undefined property 'bar'.
