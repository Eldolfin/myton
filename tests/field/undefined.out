[31merror[error]: 'Foo' object has no attribute 'bar'
----- Traceback -----
  â”Œâ”€ <unknown>:4:7
  | 
4 | foo.bar # expect runtime error: Undefined property 'bar'.
  |        ^
[0m